src/App.js

```diff
@@ -12,7 +12,16 @@ import store from './store.js';
 
 // dispatch action that will hyrate from local storage
 
+function hydrateStore(){
+	const access_token = localStorage.getItem('access_token');
+
+	if(access_token) {
```

Comment:
before logging in the user, check if the token is still valid. the token might be present in the local storage but expired.